nemo/collections/llm/recipes/gemma_2b.py

```diff
@@ -278,6 +278,9 @@ def finetune_recipe(
     recipe = default_finetune_recipe(
         model(), "google/gemma-2b", dir, name, num_nodes, num_gpus_per_node, packed_sequence
     )
+    # Gemma requires BOS
+    recipe.data.dataset_kwargs = {'add_bos': True}
+
```

Comment:
since this is shared with gemma_2b/7b, you could pull this out to a separate function that each recipe calls